windows/wiminfo.go

```diff
@@ -128,6 +128,7 @@ func DetectWindowsVersion(desc string) (version string) {
 		"2k12": {"2k12", "w2k12", "win2k12", "windows.?server.?2012"},
 		"2k16": {"2k16", "w2k16", "win2k16", "windows.?server.?2016"},
 		"2k22": {"2k22", "w2k22", "win2k22", "windows.?server.?2022"},
+		"2k25": {"2k25", "w2k25", "win2k25", "windows.?server.?2022"},
```

Comment:
Shouldn't this be the following?

```suggestion
		"2k25": {"2k25", "w2k25", "win2k25", "windows.?server.?2025"},
```